applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -505,3 +518,21 @@ def _create_line_search_strategy(self):
                                                      self.settings["move_mesh_flag"].GetBool())
         strategy.SetUseOldStiffnessInFirstIterationFlag(self.settings["use_old_stiffness_in_first_iteration"].GetBool())
         return strategy
+
+    def _create_arc_length_strategy(self):
+        settings = KratosMultiphysics.Parameters("""{}""")
+        settings.AddValue("desired_iterations", self.settings["solving_strategy_settings"]["advanced_settings"]["desired_iterations"])
+        settings.AddValue("max_radius_factor", self.settings["solving_strategy_settings"]["advanced_settings"]["max_radius_factor"])
+        settings.AddValue("min_radius_factor", self.settings["solving_strategy_settings"]["advanced_settings"]["min_radius_factor"])
+        settings.AddValue("loads_sub_model_part_list", self.settings["solving_strategy_settings"]["advanced_settings"]["loads_sub_model_part_list"])
+        settings.AddValue("loads_variable_list", self.settings["solving_strategy_settings"]["advanced_settings"]["loads_variable_list"])
+        settings.AddValue("max_iteration", self.settings["max_iteration"])
+        settings.AddValue("compute_reactions", self.settings["compute_reactions"])
+        settings.AddValue("reform_dofs_at_each_step", self.settings["reform_dofs_at_each_step"])
+        settings.AddValue("move_mesh_flag", self.settings["move_mesh_flag"])
+        solving_strategy = KratosMultiphysics.ArcLengthStrategy(self.GetComputingModelPart(),
+                                                                self._GetScheme(),
+                                                                self._GetConvergenceCriterion(),
+                                                                self._GetBuilderAndSolver(),
+                                                                settings)
```

Comment:
```suggestion
        solving_strategy = KratosMultiphysics.ArcLengthStrategy(self.GetComputingModelPart(),
                                                                self._GetScheme(),
                                                                self._GetConvergenceCriterion(),
                                                                self._GetBuilderAndSolver(),
                                                                self.settings["solving_strategy_settings"]["advanced_settings"])
```

no need to create a new set of Params
=> Imagine you add a new setting in the AL-Strategy. If you have this aux creation here then you need to modify the implementation here to!